common/src/main/java/org/wso2/carbon/testgrid/common/TestPlan.java

```diff
@@ -185,12 +200,12 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    public String getRepoDir() {
-        return repoDir;
+    public String getTestRepoDir() {
```

Comment:
It would be better to include java doc comments for all the public methods

Reply:
Fixed with f243d73cb375a67c4fb173c6c2d75a931d9411ae.